src/test/java/tests/AllSongsTests.java

```diff
@@ -0,0 +1,57 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AllSongsPage;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class AllSongsTests extends BaseTest {
+    @Test
+    public void countSongsInPlaylist() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage allSongsPage = new AllSongsPage(driver);
+
+        loginPage.login();
+        //Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+
+        homePage.clickAllSongs();
+        //String songsURL = "https://qa.koel.app/#!/songs";
+        //Assert.assertEquals(driver.getCurrentUrl(), songsURL);
+
+        int actualSongsQuantity = allSongsPage.countSongs();
+        int visibleSongsQuantity = allSongsPage.parseVisibleQuantityOfSongs(allSongsPage.visibleQuantityOfSongsAndDuration());
+
+        Assert.assertEquals(actualSongsQuantity, visibleSongsQuantity);
+    }
+
+    @Test
+    public void songsDurationValue(){
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        AllSongsPage allSongsPage = new AllSongsPage(driver);
+
+        loginPage.login();
+        //Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+
+        homePage.clickAllSongs();
+        String songsURL = "https://qa.koel.app/#!/songs";
+        Assert.assertEquals(driver.getCurrentUrl(), songsURL);

```

Comment:
**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/4e7a6f3f-43a4-4de9-a3e9-873700509692)